socorro/unittest/processor/test_mozilla_transform_rules.py

```diff
@@ -1910,6 +1910,34 @@ def test_everything_we_hoped_for(self):
         eq_(processed_crash['version'], '3.0b0')
         eq_(len(processor_meta.processor_notes), 2)
 
+    def test_with_aurora_channel(self):
```

Comment:
I don't understand what this is testing. Can you add a docstring explaining what behavior this is making sure is/isn't happening?